scripts/configure_k12_kit.cls

```diff
@@ -1,6 +1,33 @@
-//Runs install apex scripts for unmanaged contexts
+public class ArbitraryInstallContext implements InstallContext {
+    public ID organizationId(){
+        return UserInfo.getOrganizationId();
+    }
+    public ID installerId(){
+        return UserInfo.getUserId();
+    }
+    public Boolean isUpgrade(){
+        return false;
+    }
+    public Boolean isPush(){
+        return true;
+    }
+    public Version previousVersion(){
+        return null;
+    }
+}
+
+//Runs install apex scripts for any context
 public static void executeInstallApex() {
```

Comment:
Minor minor one. We might want to name this method more specifically as executeK12InstallApex. executeInstallApex sounds like a wrapper to me. However, ignore it if this is a temporary fix.